recipes/pyaez/meta.yaml

```diff
@@ -0,0 +1,90 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Ping! Also, try to use grayskull to create your recipes, see https://github.com/conda-incubator/grayskull

Reply:
I just removed the comments as well.